frontend/src/components/lib/components/BaseInput/BaseInput.styles.tsx

```diff
@@ -0,0 +1,37 @@
+import { TextField, Typography, styled, Select, IconButton } from '@mui/material';
```

Comment:
Please put this file on the right dir under components/base (this is why this folder still exist)